src/renderer/components/hotbar/hotbar-menu.tsx

```diff
@@ -43,6 +44,7 @@ export class HotbarMenu extends React.Component<Props> {
 
   render() {
     const { className } = this.props;
+    const hotbar = HotbarStore.getInstance();
     const hotbarIndex = HotbarStore.getInstance().activeHotbarIndex + 1;
```

Comment:
you don't need to get the instance again

Reply:
Oh, that's typo (from auto-complete). Thanks for catching 👍